projects/core/src/global-message/store/reducers/global-message.reducer.ts

```diff
@@ -27,14 +28,14 @@ export function reducer(
           },
         };
       } else {
-        const msgs = state.entities[message.type];
-
-        if (!msgs.includes(message.text)) {
+        const messages: Translatable[] = state.entities[message.type];
```

Comment:
it's fix for duplicate messages -there was compared objects (different references) instead of key

Reply:
What about `raw` messages itself? Can we also prevent duplication for them?